src/filesystem/api.h

```diff
@@ -68,6 +70,20 @@ class LocalizedPath {
   // FIXME: Remove when no longer required
   std::vector<std::shared_ptr<LocalizedPath>> other_localized_path;
 
+  static inline std::wstring string2wstring(const std::string& str)
```

Comment:
```suggestion
  static inline std::wstring String2Wstring(const std::string& str)
```
According to the style sheet, I suppose these functions should follow this naming convention, unless there's an exception I'm unaware of: https://google.github.io/styleguide/cppguide.html#Function_Names